lib/transport/selenium-webdriver/browserstack/automateTurboScale.js

```diff
@@ -16,21 +16,32 @@ class AutomateTurboScale extends Automate {
 
   async getBuildId() {
     try {
-      const builds = await this.sendHttpRequest({
-        url: `${this.ApiUrl}/builds?status=running`,
-        method: 'GET',
-        use_ssl: true,
-        port: 443,
-        auth: {
-          user: this.username,
-          pass: this.accessKey
+      let offset = 0;
+      var currentBuild;
+      // eslint-disable-next-line no-constant-condition
+      while (true) {
+        const builds = await this.sendHttpRequest({
+          url: `${this.ApiUrl}/builds?status=running&limit=20&offset=${offset}`,
+          method: 'GET',
+          use_ssl: true,
+          port: 443,
+          auth: {
+            user: this.username,
+            pass: this.accessKey
+          }
+        });
```

Comment:
Pull out this call into a separate function with parameters `buildEndpoint` and `offset`.